commands/info.js

```diff
@@ -0,0 +1,41 @@
+const { SlashCommandBuilder } = require('@discordjs/builders');
+const embed = require('../utils/embed.js');
+const ms = require('ms');
+const wait = require('util').promisify(setTimeout);
+
+const gitSha =
+	process.env.GIT_SHA || '0ed46333853d1aaab845c769a88845b4fd7b9dde';
+const commitMessage =
+	process.env.COMMIT_MESSAGE ||
+	`Remove Arm images, we no longer need these (#40)
+
+* Remove Arm images, we no longer need these
+* Remove strings that aren't needed`;
```

Comment:
Do we need default data here?
if we do for testing with cutiebot-dev maybe it would be clearer to instead have data that is clearly used for testing :thinking:

Reply:
Changed in [37d4691](https://github.com/Cutie-Club/cutiebot/pull/41/commits/37d46915f81d72dcda3a031270b01dfca332c0cb)